llvm/include/llvm/CodeGen/MIRYamlMapping.h

```diff
@@ -378,6 +378,7 @@ struct ScalarEnumerationTraits<TargetStackID::Value> {
     IO.enumCase(ID, "default", TargetStackID::Default);
     IO.enumCase(ID, "sgpr-spill", TargetStackID::SGPRSpill);
     IO.enumCase(ID, "scalable-vector", TargetStackID::ScalableVector);
+    IO.enumCase(ID, "scalable-pred-vector", TargetStackID::ScalablePredVector);
```

Comment:
nit: I would prefer not to abbreviate 'pred', i.e.
```suggestion
    IO.enumCase(ID, "scalable-predicate-vector", TargetStackID::ScalablePredicateVector);
```